ball.java

```diff
@@ -0,0 +1,12 @@
+public class ball {

```

Comment:
The class name `ball` should be capitalized as `Ball` to follow Java naming conventions for classes.